src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -119,8 +119,6 @@
 
   {% if extraVars.help is defined and extraVars.help%}
       <small class="form-text">{{ extraVars.help }}</small>
-    {% elseif form.vars.help is defined and form.vars.help %}
```

Comment:
Why remove this one ? You are sure it's never used ? 😄

Reply:
Tested with this rolled back using the prestashop_ui_kit.html template and it still works correctly in both ways with this PR forms. 
Though using the previous template with form_rest and not removing these lines in many cases gives two help lables instead of one when the help lables is moved to formType. 

I am not sure so will revert it back as it still works correctly with prestashop_ui_kit and form_widget rendering; while we use these for simplified forms we have no double help lables.